server/backbone/api/child.py

```diff
@@ -116,29 +110,30 @@ def complete_quest(qid, ts):
 
     **Child login required**
 
-    **Errors**:
+    **errors**:
 
-    404, Quest not found - quest does not exists, or not currently logged in child's
+    404, quest not found - quest does not exists, or not currently logged in child's
 
-    **Example Return**:
+    **example return**:
 
     .. code-block:: json
 
         {
-         "data": {
-          "completed_now": true,
-          "lvled_up": false,
-          "quest": {
-            "completed_on": "1559145134",
-            "description": "You're going on a quest, brush your teeth so you don't embarrass yourself.",
-            "due": 1559145600.0,
-            "id": 1,
-            "next_occurrence": 1559404800.0,
-            "recurring": true,
-            "reward": 12,
-            "title": "Brush your teeth",
+          "data": {
+            "completed_now": true,
+            "lvled_up": false,
+            "qst": {
+              "completed_on": "Sat, 06 Jul 2019 21:11:35 GMT",
+              "description": "You're going on a quest to save the princess, brush your teeth so you don't embarrass yourself.",
+              "due": 1559250000.0,
+              "id": 1,
+              "needs_verification": true,
+              "recurring": false,
+              "reward": 12,
+              "title": "This is the initial quest",
+              "verified": false
```

Comment:
so to clarify, if a quest has 
completed_now: true, needs_verification: true, verified: false
that would mean the quest is pending, correct?

Reply:
sorry, the comment is wrong, I renamed `verified` for `verified_on`. I'll update the comment